### PR TITLE
Meta project fixes

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -202,10 +202,13 @@ endif()
 set(CARGO_TARGET_DIR "${CMAKE_BINARY_DIR}/${BUILD_DIR}/cargo/build")
 
 # Add CMake tests for `cargo test/clippy/fmt/doc`.
-add_test(NAME cargo_tests COMMAND cargo test --features link_qt_object_files --all-targets --target-dir
-                                  ${CARGO_TARGET_DIR})
-add_test(NAME cargo_doc_tests COMMAND cargo test --features link_qt_object_files --doc --target-dir
```
Comment (ahayzen-kdab) [inline CMakeLists.txt:207]: can `--all-targets` and `--exclude` not be combined? So that it does "all except qml-meta-project"

Comment (ahayzen-kdab) [reply CMakeLists.txt:207]: Also this failed to build on Windows Qt 5 :thinking:

Comment (Be-ing) [reply CMakeLists.txt:207]: `--exclude` only works with `--workspace`, not `--all-targets`

**Comment by Be-ing** [comment]
I've tried disabling parallelization in both CMake and Cargo... I think we've stumbled on a bug in rustc :dizzy_face: Why it only happens with Qt5, I have no idea...

**Comment by ahayzen-kdab** [comment]
Fun, we might need to try on a real Windows system to see what is going on.

**Comment by ahayzen-kdab** [comment]
Lets merge this into the branch so it's get visibility in the main repo...

**Comment by Be-ing** [comment]
Okay. I'm not very inclined to use a Windows VM to figure out what's going on with that weird build error... so... :shrug:

**Pull request merged**